frontend/finance-tracker/src/validations/form-schema.ts

```
@@ -96,3 +96,47 @@ export const deleteCategoryFormSchema = editCategoryFormSchema.omit({
+    }),
+    walletId: z.coerce.string().min(1, {
+        message: "Wallet is required",
+    }),
+    amount: z.coerce
+        .string()
+        .min(1, {
+            message: "Amount is required",
+        })
+        .refine(
+            (value) => {
+                const amount = parseFloat(value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)